### PR TITLE
fix:fix response format bug

### PR DESCRIPTION
**Pull request by guaguaguaxia**

### CODE REVIEW

**Comment by guaguaguaxia** [comment]
@MattFisher please review it, i don't know how i close last same PR 😭

**Comment by MattFisher** [comment]
I'm not a maintainer of the project or anything, just another user!

**Comment by guaguaguaxia** [comment]
> I'm not a maintainer of the project or anything, just another user!

ok thanks

**Comment by MattFisher** [comment]
I've manually tested the change and it works for all the different audio response formats. I can't verify that it doesn't break any other endpoints though.

**Comment by guaguaguaxia** [comment]
@MattFisher  I thought about it for a second,i still think we should define a enum and add a function parameter to make code more clearly

**Comment by MattFisher** [comment]
@guaguaguaxia that would make sense if the Audio module was the only code that consumed the `APIRequestor` class, but it's not. `APIRequestor` is used all through the codebase, so if we were going to add an extra parameter to its `__init__` method, that would imply that all the other modules need to know about that parameter. In this case, that doesn't make sense, because the choices for that parameter are specific to the Audio transcribe function.
Other callers of `APIRequestor()` would then expect to be able to pass `response_format='text'` and get back some textual representation of the response, which isn't going to work.

**Comment by MattFisher** [comment]
Might want to roll back the whitespace changes in `openai/util.py`

**Comment by guaguaguaxia** [comment]
@MattFisher thanks. you can see this:https://github.com/openai/openai-python/issues/243#issuecomment-1457398603

**Comment by guaguaguaxia** [comment]
@MattFisher thanks. 

@logankilpatrick @mpokrass  you can see this:https://github.com/openai/openai-python/issues/243#issuecomment-1457398603

**Review by hallacy — APPROVED** [review]

**Comment by hallacy** [comment]
Looks awesome! Thanks for writing the PR!

**Comment by dusdb1** [comment]
Now, response_format is in plain text and not defaults to json format as it says in docs, right?

**Comment by guaguaguaxia** [comment]
> Now, response_format is in plain text and not defaults to json format as it says in docs, right?

No，if response_format type is json or verbose_json,it still return json,other return text

File: openai/api_requestor.py [hunk 1]
```diff
@@ -666,7 +666,10 @@ def _interpret_response_line(
                 headers=rheaders,
             )
```
Comment (Gabriel-Gotti) [inline openai/api_requestor.py:667]: keep this viewable over the misspelling line the override

**Pull request merged**